src/Components/Map/Markers/Markers.css

```diff
@@ -0,0 +1,58 @@
+.tooltiptext {
+  visibility: visible;
+  width: 100px;
+  color: #fff;
+  text-align: center;
+  border-radius: 6px;
+  position: absolute;
+  z-index: 0;
+  bottom: 125%;
```

Comment:
why is this very big value?

Reply:
I'm not sure why actually I took the Css from w3schools and when I sat it to a lower value the popup info shows on the top of the tooltip and it doesn't work as it supposed to

Reply:
Ok